### PR TITLE
Fix template files for k8s migration

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/org/issues/2589

This PR fixes the LICENSE to use the same content as of https://github.com/kubernetes/website/blob/master/LICENSE (as mandated by https://github.com/kubernetes/kubernetes-template-project#kubernetes-template-project). Also includes minor fixes to other template files.

cc @jberkus @coderanger @mrbobbytables

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners#owners_aliases
+
+aliases:
+  lwkd-maintainers:
+    - jberkus
+    - coderanger
+    - mbbroberg
```
Comment (coderanger) [inline OWNERS_ALIASES:7]: Are you sure these people are volunteering? Happy for more editors but want to make sure it's on purpose :D

**Review by coderanger — APPROVED** [review]
Overall +1

File: OWNERS_ALIASES (hunk 1 above)
Comment (nikhita) [reply OWNERS_ALIASES:7]: I don't know... can remove and add later, if needed too. :)

These folks were mentioned in the main issue body - https://github.com/kubernetes/org/issues/2589

Comment (coderanger) [reply OWNERS_ALIASES:7]: Aha, I trust Josh has spoken to them then. Or if not, easy to reverse (or not, I trust both of them to have admin access even if just as another back if both Josh and I are AFK).

Comment (jberkus) [reply OWNERS_ALIASES:7]: Matt Broberg volunteered when Contributor Comms approved taking LWKD.

Comment (jberkus) [reply OWNERS_ALIASES:7]: Bob is there as GH admin

**Pull request merged**